dss/operations/checkout.py

```diff
@@ -151,37 +150,43 @@ def process_keys(self):
                 checkout_status[_key] = bundle_internal_status
             elif _key.startswith(FILE_PREFIX):
                 temp = collections.defaultdict(blob_checkout=False, should_be_cached=False)
-                file_metadata = self._verify_get(self.handle, self.replica.bucket, _key)
+                file_metadata = self._get_metadata(self.handle, self.replica.bucket, _key)
```

Comment:
If there is no file meta_data, or the file is not found, would this actually be set to `None`? it looks like it was removed from the command above.

Reply:
Setting it to `None` is and was the original behavior.  I just renamed the function from `_verify_get` to `_get_metadata` since I thought it was clearer.

Reply:
You're right though, that's a bug (one of a few) whenever it eats any key that doesn't exist in the main bucket.  I've fixed it in `Verify` to `continue` when it hits this, but it ideally would have had tests to catch this.